src/likelihoods/negativebinomial.jl

```diff
@@ -0,0 +1,25 @@
+"""
+    NegativeBinomialLikelihood(r::Real=1.0, l=logistic)
+
+Negative binomial likelihood with number of successes `r`.
+
+```math
+    p(k|f) = \\frac{\\Gamma(k+r)}{k! \\Gamma(r)} p^r (1 - p)^k
+```
+On calling, this would return a negative binomial distribution with `r` successes and probability of success equal to `l(f)`.
```

Comment:
```suggestion
On calling, this returns a negative binomial distribution with `r` successes and probability of success equal to `l(f)`.
```

Reply:
Agree, I'll also change the analogous word in gamma.jl.